### PR TITLE
feat: add bgl-poc2

### PR DESCRIPTION
**Pull request by ccomben**
## Description

This PR introduces a new blog post, "Building Gno.land - Proof of Stake vs Proof of Contribution." This post was written by [@ccomben](https://github.com/ccomben) on January 26, 2024.

### CODE REVIEW

File: posts/2024-01-26_poc-2/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+slug: building-gnoland-poc-II
+tags: [gnoland, gnovm, tm2, PoC]
+authors: [christina]
+---
+
+# Proof of Contribution (PoC) 
+
+## II. Proof of Contribution vs Proof of Stake
+
+Proof of Stake (PoS) is a robust consensus mechanism that provides a more environmentally friendly and scalable alternative to Proof of Work (PoW) and powers most of the web3 industry today. As PoS pioneers, Cosmos technology secures hundreds of blockchain projects and billions of dollars of digital assets, and Ethereum (launched as a PoW chain in 2015) made the historic switch to PoS in 2022. According to [ethereum.org](https://ethereum.org/en/developers/docs/consensus-mechanisms/pos), PoS is “more secure, less energy-intensive, and better for implementing new scaling solutions compared to the previous proof-of-work architecture.” However, as we briefly discussed in [*What Is Proof of Contribution?*](link here), PoS has vulnerabilities that can corrupt the network over time.
```
Comment (waymobetta) [inline posts/2024-01-26_poc-2/README.md:13]: > [*What Is Proof of Contribution?*](link here)

Commenting as reminder.

File: posts/2024-01-26_poc-2/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+PoC solves this problem, ensuring the network becomes increasingly decentralized over time by being governed by a decentralized entity, GovDAO. Like standard PoA chains, PoC chains launch with a handful of validators that must be identified and trusted by the network, meaning governance is centralized at the start, and the chain achieves a low Satoshi Score. The system is about contributing and earning contribution units, which are slow to gain and require human interaction. It takes months (or years) before there are enough actors in the DAO and sufficient voting power for the chain to be considered decentralized enough, according to the Nakamoto coefficient. 
+
+PoC is thus slower to bootstrap than PoS and harder to achieve. You can think of PoC versus PoS as a marathon versus a sprint, whereby PoC starts slowly but then gains momentum over time, and PoS starts quickly but loses momentum over time (the graph below provides a visual representation of PoC versus PoS). 
+
+//image to go here
```
Comment (waymobetta) [inline posts/2024-01-26_poc-2/README.md:33]: Commenting as reminder.

File: posts/2024-01-26_poc-2/README.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+### Concluding Thoughts
+
+Beyond separating voting power from net wealth, a core component of Proof of Contribution (PoC) is its focus on long-term sustainability. PoC makes the system fairer and more sustainable, ensuring participants are aligned and take actions that benefit the community and the broader ecosystem. PoC is slower to bootstrap and harder to achieve than PoS but focuses on long-term alignment and security. 
+
+Unlike PoS, contributors receive rewards based on their contribution effort rather than how many tokens they stake. They are thus incentivized and recognized for the quality of their work, ideas, and alignment, driving participation and active engagement. Governance is allocated to the people most likely to care for the ecosystem’s long-term success—the contributors who have spent the most time working toward it.
+
+
```
Comment (waymobetta) [inline posts/2024-01-26_poc-2/README.md:51]: Add missing conclusion.

**Comment by waymobetta** [comment]
Proof of Contribution is a known term in the industry and serves as a consensus mechanism (ie, algorithmic); the way we are defining and implementing it differs from what others may already understand it to mean ([example](https://www.gemini.com/cryptopedia/blockchain-consensus-mechanism-types-of-algorithm#section-proof-of-contribution-po-c-po-co-consensus-mechanism)). Perhaps this isn't something we touch on in the blog post rather we remain aware of as we continue developing our unique flavor of it.

File: posts/2024-01-26_poc-2/README.md (hunk 2 above)
Comment (ccomben) [reply posts/2024-01-26_poc-2/README.md:33]: @waymobetta or @leohhhn  Can you add the image for me? Many thanks!

Comment (leohhhn) [reply posts/2024-01-26_poc-2/README.md:33]: Hey, sorry, I didn't forget, will do this soon.

Comment (leohhhn) [reply posts/2024-01-26_poc-2/README.md:33]: Hey @ccomben, in order to do this, the easiest way would be for you to add me as a contributor to your repo, and I'll add the image in this PR. Please look at the settings of your blog fork, and look for contributors or such, and add me with write access :)

Comment (ccomben) [reply posts/2024-01-26_poc-2/README.md:33]: OK, I added you as a collaborator. Let's see if it worked. ;)

**Review by waymobetta — APPROVED** [review]

**Pull request merged**